src/main/java/org/opensearch/security/ssl/http/netty/Netty4ConditionalDecompressor.java

```diff
@@ -17,7 +16,6 @@
 
 import org.opensearch.security.filter.NettyAttribute;
 
-@Sharable
```

Comment:
Do you know why class cannot be reused - is there something that we should have seen or tested?  I think its a pretty innocuous chance to make, but am concerned if the `Netty4HttpRequestHeaderVerifier` also needs to be new'ed every time.

Reply:
Thanks for context💯